### DISCUSSION TITLE
How to add attribution to a image or video referenced in the site

### DISCUSSION THREAD

**Discussion by spjohn85** [arthelokyo/astrowind#607] [Q&A]
I am working on a website, where I try to reference some of the existing images or videos already created by other people and wanted to provide attribution under an image/video. 

Is there a way to do this currently in the template. I could not find any such option. 

If there is no option currently, what's the best place to add such attributions ?

**Comment by massiccio** [comment]
Perhaps you can add a note to the footer?

**Reply by spjohn85** [reply]
I could do that, but I wanted to provide something under each image or video I am referencing so that it's quite visible to users. I don't want to mislead users into thinking that it's my own work.
